### ISSUE TITLE
unused-import false positive

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#304]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

For the given code. It should warn only for OrderedDict.

```
#!python

from collections import deque, OrderedDict
deque()
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/304
